src/reaper_osara.cpp

```diff
@@ -2068,6 +2078,13 @@ PostCustomCommand POST_CUSTOM_COMMANDS[] = {
 	{"_XENAKIOS_NUDGEITEMVOLUP", postChangeItemVolume}, // Xenakios/SWS: Nudge item volume up
 	{NULL},
 };
+
+using MExplorerPostExecute = void (*)(int, HWND);
+map<int, MExplorerPostExecute> MExplorerPostCommands{
```

Comment:
nit: The variable name should start with a lower case letter to differentiate variables from custom types:
```suggestion
map<int, MExplorerPostExecute> mExplorerPostCommands{
```